spec/features/users/user_login_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'User Login Spec' do
+  describe 'When I visit the login page' do
+    describe 'I can log in with valid credentials email and password' do
+
+      it 'If I am a regular user, I am redirected to my profile page. A flash message will confirm I am logged in' do
+        user = create(:user, role: 0)
+        visit '/login'
+        fill_in :email, with: user.email
+        fill_in :password, with: user.password
+        click_on 'Log In'
+        expect(current_path).to eq("/profile")
+        expect(page).to have_content("You are now logged in!")
+      end
+
+      it 'If I am a merchant user, I am redirected to my merchant dashbpard page. A flash message will confirm I am logged in' do
```

Comment:
spell check: dashbpard to **dashboard** 
Line 17 of `spec/features/users/user_login_spec.rb`